flight/apps/comms/comms.py

```diff
@@ -37,20 +37,23 @@ class MSG_ID:
 
 
 class SATELLITE_RADIO:
+    # Hardware abstraction for satellite
+    sat = SATELLITE
```

Comment:
would remove that reference and use directly SATELLITE in this file. This would be clear that this comes from the HAL